idea/resources/META-INF/plugin-common.xml

```diff
@@ -3346,8 +3346,8 @@
                      language="kotlin"
     />
 
-    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.ReplaceJavaIntegerToStringWithMemberInspection"
-                     displayName="Replace 'Integer.toString(int)' with 'int.toString()'"
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.ReplaceJavaNumberToStringWithMemberInspection"
+                     displayName="Replace 'Number.toString(int)' with 'number.toString()'"
```

Comment:
I find `Number.toString(int)` here slightly confusing, because `Number` class doesn't have such method. Also `int` parameter would be confusing for other numeric types.

Perhaps `Replace static 'toString(number)' with 'number.toString()'` would read better?